python-sdk/tutorials/automl-with-azureml/automl-nlp-multiclass/automl-nlp-text-classification-multiclass.ipynb

```diff
@@ -0,0 +1,678 @@
+    "test_data_fname = data_dir + \"/test_data.csv\"\n",
+    "\n",
+    "data_train.to_csv(train_data_fname, index=False)\n",
+    "data_val.to_csv(val_data_fname, index=False)\n",
+    "data_test.to_csv(test_data_fname, index=False)\n",
+    "\n",
+    "datastore = ws.get_default_datastore()\n",
+    "datastore.upload(src_dir=data_dir, target_path=blobstore_datadir, overwrite=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1633480048150
+    }
+   },
+   "outputs": [],
+   "source": [
+    "train_dataset = Dataset.Tabular.from_delimited_files(\n",
+    "    path=[(datastore, blobstore_datadir + \"/train_data.csv\")]\n",
+    ")\n",
+    "val_dataset = Dataset.Tabular.from_delimited_files(\n",
+    "    path=[(datastore, blobstore_datadir + \"/val_data.csv\")]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1633480049084
+    }
+   },
+   "outputs": [],
+   "source": [
+    "train_dataset = train_dataset.register(\n",
+    "    workspace=ws, name=\"20newsgroups_data_train\", description=\"20newsgroups_data_train\"\n",
+    ")\n",
+    "\n",
+    "val_dataset = val_dataset.register(\n",
+    "    workspace=ws, name=\"20newsgroups_data_val\", description=\"20newsgroups_data_val\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Train\n",
+    "\n",
+    "## Submit AutoML run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can start the run with the prepared compute resource and datasets. This should only take a few minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1633480049334
+    }
+   },
+   "outputs": [],
+   "source": [
+    "automl_settings = {\n",
+    "    \"iterations\": 1,\n",
+    "    \"primary_metric\": \"accuracy\",\n",
+    "    \"max_concurrent_iterations\": 1,\n",
+    "    \"featurization\": \"auto\",\n",
+    "    \"verbosity\": logging.INFO,\n",
+    "}\n",
+    "\n",
+    "automl_config = AutoMLConfig(\n",
+    "    task=\"text-classification\",\n",
+    "    debug_log=\"automl_errors.log\",\n",
+    "    compute_target=compute_target,\n",
+    "    training_data=train_dataset,\n",
+    "    validation_data=val_dataset,\n",
+    "    label_column_name=target_column_name,\n",
+    "    **automl_settings\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Submit AutoML Run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1633480498717
+    }
+   },
+   "outputs": [],
+   "source": [
+    "automl_run = experiment.submit(automl_config, show_output=False)\n",
+    "_ = automl_run.wait_for_completion(show_output=False)"
```

Comment:
Is there any reason to assign the return value from wait_for_completion?
I don't see that in other notebooks.